packages/phone-number-privacy/combiner/package.json

```diff
@@ -24,9 +24,9 @@
     "db:migrate": "NODE_ENV=dev ts-node scripts/run-migrations.ts"
   },
   "dependencies": {
-    "@celo/contractkit": "1.0.2",
+    "@celo/contractkit": "1.2.2-dev",
```

Comment:
Unfortunately the combiner cannot rely upon an unpublished version. Google functions won't allow it.

Reply:
Should I change version in all places to something published but still with cEUR in contractkit ?